client/state/plans/get-interval-type-for-term.js

```diff
@@ -0,0 +1,17 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+
+import { TERM_ANNUALLY, TERM_BIENNIALLY, TERM_MONTHLY } from 'lib/plans/constants';
+
+export default function getIntervalTypeFromTerm( term ) {
+	return (
+		{
```

Comment:
This will create a new object every time we call this function. I'd suggest defining it separately above the `export`, and using it here.

Another thing you could do is use a `switch` clause and completely avoid creating an object.

Reply:
> This will create a new object every time we call this function. I'd suggest defining it separately above the `export`, and using it here.

True, and I've actually recommended and made changes like those myself. On the other hand, folks like @jsnajdr have been encouraging lazy instantiation for cases just like this to try to keep initial boot as speedy as possible (correct me if I'm wrong). The decision here probably has very limited impact and would be a great place to bikeshed and over-engineer a solution 😵 except…

> Another thing you could do is use a switch clause and completely avoid creating an object.

This would be a trivial change that covers requirements nicely.